tests/test_uwsgi_plugin.py

```diff
@@ -0,0 +1,62 @@
+from testing.sandbox import install_fake_uwsgi
+install_fake_uwsgi()
```

Comment:
@asottile no you're definitely right, but they spin it up and query against the provided socket, but they don't actually run the test *inside* the server, so I don't think they'd have access to the module. I'll poke around with it though, because it really would be great to have a solid integration test for this.